src/BigBlueButton.php

```diff
@@ -524,6 +549,21 @@ public function setJSessionId($jSessionId)
      * @throws \RuntimeException
      */
     private function processXmlResponse($url, $payload = '', $contentType = 'application/xml')
+    {
+        return new SimpleXMLElement($this->processResponse($url, $payload, $contentType));
+    }
+
+    /**
+     * A private utility method used by other public methods to process responses.
+     *
+     * @param string $url
+     * @param string $payload
+     * @param string $contentType
+     *
+     * @return string            Response body
+     * @throws \RuntimeException
+     */
+    private function processResponse($url, $payload = '', $contentType = 'application/xml')
     {
         if (extension_loaded('curl')) {
```

Comment:
Can we negate this and throw the error at this point? The remaining function would be a bit more readable.

Reply:
@sualko If you are ok with the latest commit, resolve this conversation.